### PR TITLE
Remove variableScope warnings reported by cppcheck

### PR DESCRIPTION
**Pull request by rbuj**
```
cppcheck --enable=all . 2> err.txt
grep variableScope err.txt
```

### CODE REVIEW

File: applets/clock/clock-location.c [hunk 1]
```diff
@@ -76,10 +76,11 @@ clock_location_find_and_ref (GSList       *locations,
                              const gchar *code)
 {
         GSList *l;
-        ClockLocationPrivate *priv;
 
         for (l = locations; l != NULL; l = l->next) {
-                priv = clock_location_get_instance_private (l->data);
+
+                ClockLocationPrivate *priv =
+                        clock_location_get_instance_private (l->data);
```
Comment (sc0w) [inline applets/clock/clock-location.c:83]: why not in the same line?

**Comment by sc0w** [comment]
as @cwendling mentioned in https://github.com/mate-desktop/mate-panel/pull/1126/files#r465025529 , for a better code readability, at some points is better to separate the declaration from the assignation

**Comment by rbuj** [comment]
> as @cwendling mentioned in https://github.com/mate-desktop/mate-panel/pull/1126/files#r465025529 , for a better code readability, at some points is better to separate the declaration from the assignation

PR #1130 by @cwendling.

https://github.com/mate-desktop/mate-panel/blob/7d706e9fe1f53ef73f644eb1a6488239541d6714/mate-panel/panel.c#L612

https://github.com/mate-desktop/mate-panel/blob/7d706e9fe1f53ef73f644eb1a6488239541d6714/mate-panel/panel.c#L616-L617

@sc0w making cross-references is a bad practice... @cwendling I'm sorry for the inconvenience.

File: applets/clock/clock-location.c (hunk 1 above)
Comment (rbuj) [reply applets/clock/clock-location.c:83]:
```
$ echo -ne '                ClockLocationPrivate *priv = clock_location_get_instance_private (l->data);' | wc 
      0       5      91
```

**Comment by lukefromdc** [comment]
The travis failure was just a failure of git submodule to work properly, cause by failure to access the server:
`unable to access 'https://github.com/mate-desktop/mate-submodules.git` That has nothing to do with the code changes here so should not affect evaluation of the code changes

**Comment by raveit65** [comment]
What a huge work for variable Scope warnings.
I will test this large commit a bit.

**Review by raveit65 — APPROVED** [review]
LGTM,
i didn't noticed any issue with the panel the last days.

**Pull request merged**